### DISCUSSION TITLE
Simkl integration

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [pymedusa/Medusa#8876] [Ideas & Feature requests]
I want to ask to integrate Simkl with Medusa. Trakt.tv don't work from some time and cannot sync library with Medusa. I also have problem to sync it with my Emby Server. Simkl has Emby, Plex and others plugins that works just fine.
Simkl is also track shows and movies and looks really great. They have very nice smartphone app. Dashboard is very clean and beautiful. It's not as poor as the trakt.tv in the free version.
https://simkl.com/

Regards,
Przemek

**Comment by masyk** [comment]
I would like to see Simkl support as well.

**Reply by AubreyKloppers** [reply]
I would also like to see this feature.  Radarr has already been added, so I would seriously like to see Simkl into Medusa as well...

**Comment by Azmarine13** [comment]
Any update on this? Waiting for Sync feature update

**Comment by Urkov** [comment]
Any plan to integrated?

**Comment by utamayo** [comment]
Trakt now has limited a lot free accounts 😣 so it will be great if someone could add integration with Simkl.

It will be a new year's wish...😊
